### ISSUE TITLE
Recalculate resistor value

### ISSUE THREAD

**Issue by marove2000** [Binary-Kitchen/kitchen-smd-challange#3]
Some LEDs are too bright, some are too faint.

**Comment by ventilator** [comment]
I made some measurement on a baseline of I=0.9mA for the red LED, then tuned the other LEDs to the same brightness. Brightness was measured with my eyes, so it is a bit subjective.

  | Measurement |   |   | |Calculated  |  E12 
-- | -- | -- | -- | -- | -- | --
  | U (V) | I (mA) | R now (Ohm) | U r (V) | R future (Ohm) |  
rot | 1,80 | 0,9 | 82 | 1,20 | 83 | 82 | baseline
orange | 1,86 | 0,9 | 82 | 1,14 | 79 | 82 |  
gelb | 1,91 | 1,1 | 100 | 1,09 | 62 | 62 |  
grün | 2,15 | 0,16 | 150 | 0,85 | 332 | 330 | very bright led
blue | 2,57 | 0,9 | 200 | 0,43 | 30 | 33 |  
  |   |   |   |   |   |   |  
  |   |   |   |   |   |   |  
Battery | 3 | V |   |   |   |   |  
Scaling factor | 16 |   |   |   |   |   |  



Scaling factor means   that measurement was done at steady state, but LED flashes in application

Basically I scaled   the 0,9mA baseline current for red to the acutal 82 Ohm used in application


Anyway, I recommend resistors as shown in the last column, rounded to the E12 line

**Comment by marove2000** [comment]
fixed. Thank you for the calculation.
